ArduCopter/mode_auto.cpp

```diff
@@ -1217,10 +1217,10 @@ void ModeAuto::nav_attitude_time_run()
     }
 
     // constrain climb rate
-    float target_climb_rate_cms = constrain_float(nav_attitude_time.climb_rate * 100.0, pos_control->get_max_speed_down_cms(), pos_control->get_max_speed_up_cms());
+    float target_climb_rate_ms = constrain_float(nav_attitude_time.climb_rate, pos_control->get_max_speed_down_ms(), pos_control->get_max_speed_up_ms());
```

Comment:
again not a blocker but at some point we should consider renaming nav_attitude_time.climb_rate to nav_attitude_time.climb_rate_m unless we consider that we're already beyond the point where "m" has become the standard